hw_diag/diagnostics/add_gateway_txn_diagnostic.py

```diff
@@ -0,0 +1,42 @@
+import json
+import secrets
+
+from hm_pyhelper.diagnostics import Diagnostic
+from hm_pyhelper.constants.shipping import DESTINATION_WALLETS_KEY
+from hm_pyhelper.miner_json_rpc.exceptions import MinerMalformedAddGatewayTxn
+
+from hw_diag.utilities.miner import create_add_gateway_txn
+from hw_diag.utilities.security import get_verified_data
+
+KEY = 'ADD_GATEWAY_TXN'
+FRIENDLY_NAME = "add-gateway-transaction"
+
+NO_DESTINATION_WALLETS_MSG = "Destination wallets not found in payload JSON."
```

Comment:
If I understand @kashifpk, `NO_DESTINATION_WALLETS_MSG` and `VERIFICATION_FAILED_MSG` should be class variable since they won't be accessed externally.

Reply:
Yes @marvinmarnold - That's just to avoid namespace collisions if the same name gets used elsewhere and we import both of these in a single module. @posterzh placing them inside the class  ensures that they can't be  imported to global namespace with `from add_gateway_txn_diiagnostic import *`. Although using `from module import *` isn't something we do or should do.

This is being just a bit more careful :-)

Reply:
Make sense.